public/app/features/transformers/timeSeriesTable/timeSeriesTableTransformer.ts

```diff
@@ -0,0 +1,125 @@
+import { map } from 'rxjs/operators';
+
+import {
+  ArrayVector,
+  DataFrame,
+  DataTransformerID,
+  DataTransformerInfo,
+  Field,
+  FieldType,
+  MutableDataFrame,
+} from '@grafana/data';
+
+export interface TimeSeriesTableTransformerOptions {}
+
+export const timeSeriesTableTransformer: DataTransformerInfo<TimeSeriesTableTransformerOptions> = {
+  id: DataTransformerID.timeSeriesTable,
+  name: 'Time series to table transform',
+  description: 'Time series to table rows',
+  defaultOptions: {},
+
+  operator: (options) => (source) =>
+    source.pipe(
+      map((data) => {
+        return timeSeriesToTableTransform(options, data);
+      })
+    ),
+};
+
+/*
+For each refId (queryName) convert all time series frames into a single table frame, adding each series
```

Comment:
It's awesome to see a comment like this 🥳. Having [TSDoc](https://tsdoc.org/) style would be even better

Reply:
rewrote to use TSDoc style :)